packages/office-ui-fabric-react/src/utilities/selectableOption/SelectableDroppableText.Props.ts

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { IRenderFunction } from '../../Utilities';
+import { ICalloutProps } from '../../Callout';
+import { ISelectableOption } from '../../utilities/selectableOption/SelectableOption.Props';
+
+export interface ISelectableDroppableTextProps<T> extends React.Props<T> {
+  /**
+  * Optional callback to access the ISelectableDroppableText interface. Use this instead of ref for accessing
+  * the public methods and properties of the component.
+  */
+  componentRef?: (component: T) => void;

```

Comment:
this is used above like: ISelectableDroppableTextProps<IComboBoxProps>, so the componentReference is really of type IComboBoxProps, is that the intention?